e2e/test.js

```diff
@@ -0,0 +1,51 @@
+const expect = require('chai').expect;
+
+describe('InstantSearch e-commerce demo', () => {
+  before(() => {
+    browser.maximizeWindow();
+  });
+
+  it('must work', async () => {
+    await browser.url(
+      'https://deploy-preview-3967--instantsearchjs.netlify.com/examples/e-commerce/'
+    );
+
+    // Select "Apple" brand in list
+    const brand = await browser.$('span=Apple');
+    await brand.click();
+
+    // Fill search input with "macbook"
+    const searchInput = await browser.$('[type=search]');
+    await searchInput.setValue('macbook');
+
+    // Wait for the results list to be updated (wait for the "macbook" word to be highlighted)
+    await browser.waitUntil(
+      async () => (await browser.$$('mark=MacBook')).length > 0,
+      10000
+    );
+
+    // Get title for each result
+    const hits = await browser.$$('.hit h1');
+    const hitsText = await Promise.all(hits.map(hit => hit.getText()));
```

Comment:
Why do we extract only the text and not the HTML for also the highlighted parts? This seems to cover better the actual usage.

Reply:
Here I'm only testing that I got the right list of results, I do not test that the text is correctly highlighted.